src/executePrettier.js

```diff
@@ -37,7 +47,15 @@ const executePrettier = (editor, text) => {
           filePath: getCurrentFilePath(editor),
         }));
     }
-    return prettier.format(text, getPrettierOptions(editor));
+
+    const prettierOptions = getPrettierOptions(editor);
+    const localPrettier = getLocalPrettierPath(getCurrentFilePath(editor));
+
+    if (!localPrettier) {
+      return prettier.format(text, prettierOptions);
+    }
+
+    return getLocalPrettier(localPrettier).format(text, prettierOptions);
```

Comment:
Sure, that makes more sense, we can then just past the prettier instance to use to the `executePrettier` function. 

We could even pass prettier down the execute stack from `format` / `formatOnSave`, like the `editor`.